api-client/src/main/java/com/xing/api/CallSpec.java

```diff
@@ -302,9 +302,10 @@ private Call createRawCall() {
     @Nullable
     private <PT> PT parseBody(Type type, ResponseBody body) throws IOException {
         if (body == null) return null;
-        if (Types.getRawType(type) == Void.class) return null;
         BufferedSource source = body.source();
         try {
+            // Don't consume the body, if the user expects a void.
+            if (Types.getRawType(type) == Void.class) return null;
```

Comment:
Squeezing this in. This makes sure that the body will be closed, even if not parsed.